tasks/config/licenses.js

```diff
@@ -32,6 +32,7 @@ module.exports = function (grunt) {
         'UNLICENSE'
       ],
       overrides: {
+        'abbrev@1.0.7': ['MIT'],
```

Comment:
Cool, as long as you add the correct license to the overrides we can avoid calling Septa Unella

![](http://imagesmtv-a.akamaihd.net/uri/mgid:file:http:shared:mtv.com/news/wp-content/uploads/2016/06/shame-gif-1465520937.gif)